apps/src/sites/studio/pages/lti/v1/_early_access_banner.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * @file Inits the LtiEarlyAccessBannerHandler on the LTI Early Access banner load.
+ * @see dashboard/app/views/lti/v1/_early_access_banner.html.haml
+ */
+import getScriptData from '@cdo/apps/util/getScriptData';
+
+class LtiEarlyAccessBannerHandler {
```

Comment:
This should be a React Functional Component. As an engineering org, we are slowly migrating from using `class` components. We are also starting to adopt TypeScript. For example:
https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/lib/ui/lti/sync/LtiSectionSyncDialog.tsx